target/linux/ramips/image/Makefile

```diff
@@ -21,10 +21,12 @@ KERNEL_LOADADDR := $(loadaddr-y)
 
 KERNEL_DTB = kernel-bin | patch-dtb | lzma
 define Device/Default
+  ATH_SOC :=
```

Comment:
ATH actually means Atheros. You might want RT_SOC (RT for Ralink) or MT_SOC (MT for Mediatek, who acquired Ralink) here.

Reply:
And a new variable in device template should be added to DEVICE_VARS.

Reply:
Oh..... Yes.

Reply:
I suggest ```MTK_SOC```.

Reply:
Oops...
Variables in DEVICE_VARS will be cleared for every single device profile. If it's defined at the beginning of {subtarget}.mk, it might not be a good idea to put ATH/RT/MT_SOC in device profile.
This may need to be defined for every devices since we have mt7620a/mt7620n in mt7620, mt7628/mt7688 in mt76x8 and so on.

Reply:
Per-device would then be the same as for ath79.
This looks a little undesirable to me (as subtargets are already "sorted"), but I do not see a way around yet.

Reply:
I've now put MTK_SOC in device profiles for all subtargets. This is easy to achieve programatically, and if someone finds a way to specify a common subtarget definition, it would also be easy to remove again.